src/content/docs/new-relic-solutions/get-started/glossary.mdx

```diff
@@ -23,7 +23,7 @@ redirects:
 
 import accountsAccountSwitcher from 'images/accounts_screenshot-crop_account-switcher.webp'
 
-Whether you're considering New Relic or you're already using our capabilities, this glossary of common terminology can help. And if you don't already have a New Relic account, don't hesitate to sign up at [newrelic.com/signup](https://newrelic.com/signup). It's free, forever!
+Whether you're considering New Relic or you're already using our capabilities, this glossary of common terminology can help. And if you're looking to get started, see [Start using New Relic](/docs/new-relic-solutions/get-started/intro-new-relic/).
```

Comment:
As a note (not action), wondering how tutorial squad and spruce up the get started/intro doc. Is it worth using as a landing page for our tutorial content?